definitions/json/EnglandWales Scrubbed/EnglandWales Scrubbed.json

```diff
@@ -728,13 +728,13 @@
   {
     "ID": "fl_HearingPanelPreference",
     "ListElementCode": "Judge",
-    "ListElement": "Judge",
+    "ListElement": "I prefer my case to be heard by a judge",
```

Comment:
I'm pretty sure these labels are used elsewhere for something else so it would be better to create a new fixed list item like fl_HearingPanelPreferenceSentence. Same for the change below as well